sympy/matrices/eigen.py

```diff
@@ -119,7 +120,8 @@ def _eigenvals(
     =======
 
     eigs : list or dict
-        Eigenvalues of a matrix. The return format would be specified by
+        key: Eigenvalues of a matrix. value:  multipilicity of the eigenvalue.
```

Comment:
This is not very accurate documentation because it does not apply for `list` case.
To be more friendly, I'd rather suggest using more explicit description for them:
```
The return format is either
- A dictionary ``{k1: v1, k2: v2, ...}`` where ``k1, k2, ...`` are the eigenvalues and ``v1, v2, ...`` are their respective multiplicities.
- A list ``[v1, v2, ...]`` where ``v1, v2, ...`` are potentially duplicate list of eigenvalues of the matrix.
```